test/check-machines-storage-pools

```diff
@@ -695,6 +695,59 @@ class TestMachinesStoragePools(VirtualMachinesCase):
             remove=True,
         ).execute()
 
+    def testGeneralCheckForPoolsDeletion(self):
```

Comment:
The biggest part of this test can be removed as fix for my next comment.. The remaining code can be merged with the existing test for pools deletion.